libr/bin/format/elf/elf.c

```diff
@@ -3390,6 +3427,16 @@ static RBinElfSymbol* Elf_(_r_bin_elf_get_symbols_imports)(ELFOBJ *bin, int type
 	Elf_(Sym) *sym = NULL;
 	ut8 s[sizeof (Elf_(Sym))] = { 0 };
 	char *strtab = NULL;
+	HtPP *symbol_map = NULL;
+	HtPPOptions symbol_map_options = {
+		.cmp = (HtPPListComparator)cmp_RBinElfSymbol,
+		.hashfn = hashRBinElfSymbol,
+		.dupkey = NULL,
+		.calcsizeK = NULL,
```

Comment:
you may want to implement calcsizeK as well (maybe just use the strlen of the name symbol) as it is used as a quick check to see if two keys are equal.

Reply:
I can try it, but I doubt it will give much improvement on top offset check which is done before any string comparisons.

Reply:
good point ;) ok. How is it going btw? Is the test binary still fast to analyze with this change to Ht?

Reply:
There is no noticeable difference, with and without strlen.

Reply:
Good. Do you have other changes to do?

Reply:
No